ToyBox/Classes/Models/Settings.cs

```diff
@@ -24,9 +24,12 @@ public class PerSaveSettings : EntityPart {
         //      Dictionary<CharacterName, 
         //          Dictionary<ClassID, HashSet<ArchetypeIDs>
 
+        // Dictionary<Character Hashcode, Dictionary<Dictionary<bodyPartName, universalParameter>, Dictionary<bodyPartName, sizeMultiplier>>
+        [JsonProperty]
+        public Dictionary<string, Dictionary<string, float>> characterSkeletonReplacers = new();
         // This is the scaling modifier which is applied to the visual model of each character
         [JsonProperty]
-        public Dictionary<string, float> characterModelSizeMultiplier = new();
```

Comment:
Just changing this could cause issues, least of all that existing settings will be made useless.
Have you thought about adding a new setting instead of renaming the current setting? When you then load the new setting, you can check for the old setting, and if it exists port it over to the new format, i.e:
1. Create (old, 1, 1); (I think the old one was just scaling in x dimension; though I'm not sure.)
2. Save the new one 
3. Delete the old one

Reply:
We can revert everything back to how it was. I actually started modifying the mod with this approach, thinking it was a good idea. However, I later realized that using transform scaling is not the best option for resizing the character because it breaks animations. Transform scaling applies a scale multiplier to all skeleton bones at once, including the root bone and the position bone. As a result, any animations (e.g., sitting on a throne) where the position does not align with the root bone break, because the distance between the root bone and the position bone is multiplied by the root bone's scale.

I’ve added the ability to resize the character by scaling the pelvis bone instead. So, ultimately, this change isn’t that critical.